db/data.py

```diff
@@ -359,6 +360,58 @@ def load_low_level(mbid, offset=0):
         return row[0]
 
 
+def load_many_low_level(recordings):
+    """Recordings is a list of tuples (mbid, offset).
```

Comment:
We have some guidelines here regarding how to write docstrings: https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings

Specifically, I wouldn't have the "Steps" section below, or the text starting "Sample query response..."

Reply:
Thanks for the link! I've read and bookmarked those guidelines, and have altered the docstring accordingly.